html/header.php

```diff
@@ -0,0 +1,36 @@
+<header>
+    <link rel="stylesheet" href="/css/header.css"/>
+</header>
```

Comment:
Hier bitte den header von Luisa importen.

Reply:
Muss doch nicht, oder? Der Header wird ja nie alleine verwendet nur in andere Seiten eingebunden. Und diese Seiten haben ja die Metadaten drin